src/test/java/uk/gov/pay/ledger/event/model/TransactionEntityFactoryTest.java

```diff
@@ -71,7 +72,6 @@ public void fromShouldConvertEventDigestToTransactionEntity() {
         assertThat(transactionEntity.getTransactionType(), is("PAYMENT"));
         assertThat(transactionEntity.isLive(), is(true));
 
-
```

Comment:
Don't we need assertion removed for not null source ? `assertThat(transactionEntity.getSource().toString(), is("CARD_API"));`